sites/README.md

```diff
@@ -0,0 +1,293 @@
+
+### image
+**(Mandatory)**
+
+**Type:** String
+
+Set the Docker image you want to use.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+```
+
+### tag
+
+**Type:** String
+
+**Default:** latest
+
+You can specify the tag you want to use with the image.
+
+Example:
+```
+container:
+    image: prod-comms.docker-registry.canonical.com/canonical.com
+    tag: a264efb326485
+```
+
+## wwwPrefixHost
+
+**Type:** Boolean
+
+**Default:** True
+
+When True it will include an extra host with the www prefix to the Ingress object.
+
+Example:
+```
+wwwPrefixHost: false
+```
+
+## tlsSecretName
+
+**Type:** String
+
+**Default:** {name}-tls
+
+Here you can define the name of the TLS secret for the domain. This name will be used for the www prefix host if `wwwPrefixHost` is True.
+
+Example:
+```
+tlsSecretName: cloud-init-org-tls
+```
+
+## useSquid
+
+**Type:** Boolean
+
+**Default:** False
+
+This will use our ConfigMap object "proxy-config", you can take a look to this object here: `configmaps/proxy-config.yaml`
+
+Example:
+```
+useSquid: True
+```
+
+## readinessPath
+
+**Type:** String
+
+**Default:** /_status/check
+
+Define the URL path to do a readiness probe.
+
+Example:
+```
+readinessPath: /
+```
+
+## readinessPeriodSeconds
+
+**Type:** Int
+
+**Default:** 5
+
+How often (in seconds) to perform the probe. Default to 5 seconds. Minimum value is 1.
+
+Example:
+```
+readinessPeriodSeconds: 10
+```
+
+## readinessSuccessThreshold
+
+**Type:** Int
+
+**Default:** 1
+
+Minimum consecutive successes for the probe to be considered successful after having failed. Defaults to 1. Must be 1 for liveness. Minimum value is 1.
+
+Example:
+```
+readinessSuccessThreshold: 5
+```
+
+## readinessTimeoutSeconds
+
+**Type:** Int
+
+**Default:** 3
+
+Number of seconds after which the probe times out. Defaults to 3 second. Minimum value is 1.
+
+Example:
+```
+readinessTimeoutSeconds: 5
+```
+
+## memory
+
+**Type:** String
+
+**Default:** 256Mi
+
+It will define the container memory limit.
+
+Example:
+```
+memory: 512Mi
+```
+
+## envSecrets
+
+This property is for environment related configurations. Here you can set the [name](#name-1) and the [value](#value) or [secretKeyRef](#secretKeyRef).
+
+### name
+
+**Type:** String
+
+Example:
+```
+envSecrets:
+  - name: SENTRY_DSN
+```
+
+### value
+
+**Type:** String
+
+Example:
+```
+envSecrets:
+  - name: SENTRY_DSN
+    value: https://426397ba83be483a8a8d1ed92b0f0623@sentry.is.canonical.com//17
+```
+
+### secretKeyRef
+#### key & name
+
+**Type:** String
+
+Example:
+```
+  - name: SEARCH_API_KEY
+    secretKeyRef:
+      key: google-custom-search-key
+      name: google-api
+```
+
+## extraHosts
+### host
+
+**Type:** Array
+
+Domains to be included in the Ingress object.
+
+Example:
+```
+extraHosts:
+  - host:
+    - docs.staging.jujucharms.com
+```
+
+### tlsSecret
```

Comment:
We want the name of the TLS secrets to be absolutely consistent, and to be derived from the domain. For `ubuntu.com`, it's `ubuntu-com-tls`. For this reason, I again don't think this should be a setting. It should be computed from the domain name.

Reply:
Removed